test/e2e/app-dir/ppr-metadata-streaming/ppr-metadata-streaming.test.ts

```diff
@@ -0,0 +1,39 @@
+import { nextTestSetup } from 'e2e-utils'
+
+describe('ppr-metadata-streaming', () => {
```

Comment:
It's not completely clear to me what the change in behavior is: afaict this test suite is passing on `canary` as well.
![CleanShot 2025-01-28 at 17 47 40@2x](https://github.com/user-attachments/assets/f86f3623-5e15-4866-8507-16bc0e2fb561)